tools/appveyor.psm1

```diff
@@ -326,9 +326,15 @@ function Invoke-AppVeyorTest
     Write-Host -Foreground Green 'Upload FullCLR test results'
     Update-AppVeyorTestResults -resultsFile $testResultsFileFullCLR
 
-    ## CodeCoverage
-    $env:CodeCoverageOutput = Split-Path -Parent (Get-PSOutput -Options (New-PSOptions -Configuration CodeCoverage))
- 
+    if(Test-DailyBuild)
+    {
+        ## Publish code coverage build, tests and OpenCover module to artifacts, so webhook has the information.
+        ## Build webhook is called after 'after_test' phase, hence we need to do this here and not in AppveyorFinish. 
+        $CodeCoverageOutput = Split-Path -Parent (Get-PSOutput -Options (New-PSOptions -Configuration CodeCoverage))
```

Comment:
Minor: maybe change `$CodeCoverageOutput` to `$codeCoverageOutput` with lower `c`, so that it's in the same format as the variables in the context.